github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits-1.49.yaml

```diff
@@ -0,0 +1,179 @@
+presubmits:
+  kubevirt/containerized-data-importer:
+  - name: pull-cdi-unit-test-1.49
+    branches:
+      - release-v1.38
+      - release-v1.49
+    cluster: prow-workloads
```

Comment:
I tried to add context: but it failed when I had two different jobs with the same context, not sure what's that about.

Reply:
According to [test-infra](https://github.com/kubernetes/test-infra/blob/f6c02074bb4274351972b7313f64de61ab7bcd87/prow/config/jobs.go#L323-L322) `Context` points to the name of the GitHub status context. `Name` OTOH points to how the job itself is named.

Each job definition needs to have a unique value for the `name`, and that is true for all job configs that live inside a prow config, beyond repositories or orgs boundaries.

OTOH the mapping to the GitHub status context should be kept always the same, as in one same value for all jobs doing the same thing but for different branches.

Reply:
Instead of waiting for the config-checker to validate whether everything is ok, you can also use config-checker from test-infra yourself:

```
~/Projects/github.com/kubevirt.io/project-infra $ podman run -v $(pwd):/tmp/config:Z -it --rm gcr.io/k8s-prow/checkconfig:v20221108-d3ce231cb9 --config-path=/tmp/config/github/ci/prow-deploy/files/config.yaml --job-config-path /tmp/config/github/ci/prow-deploy/files/jobs/
{"component":"checkconfig","error":"dialing: google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.","file":"k8s.io/test-infra/prow/io/opener.go:139","func":"k8s.io/test-infra/prow/io.createGCSClient","level":"debug","msg":"Cannot load application default gcp credentials, falling back to anonymous client","severity":"debug","time":"2022-11-09T11:18:04Z"}
{"component":"checkconfig","file":"k8s.io/test-infra/prow/cmd/checkconfig/main.go:252","func":"main.main","level":"info","msg":"checkconfig passes without any error!","severity":"info","time":"2022-11-09T11:18:04Z"}
```

Reply:
Looks good now -- the problem must have been that I left a few "release-v1.38" references in files other than the 1.38-specific files, so they were duplicates.